clang/test/Driver/sycl-spirv-ext-old-model.c

```diff
@@ -24,6 +24,63 @@
 // RUN:  | FileCheck %s -check-prefixes=CHECK-DEFAULT
 // RUN: %clang -target x86_64-unknown-linux-gnu -fsycl --no-offload-new-driver -fsycl-targets=spir64_x86_64-unknown-unknown %s -### 2>&1 \
 // RUN:  | FileCheck %s -check-prefixes=CHECK-CPU
+// RUN: %clang -target x86_64-unknown-linux-gnu -fsycl --no-offload-new-driver -fsycl-targets=spir64_x86_64-unknown-unknown -fno-offload-fp32-prec-div %s -### 2>&1 \
+// RUN:  | FileCheck %s -check-prefixes=CHECK-CPU
+// RUN: %clang -target x86_64-unknown-linux-gnu -fsycl --no-offload-new-driver -fsycl-targets=spir64_x86_64-unknown-unknown -fno-offload-fp32-prec-sqrt %s -### 2>&1 \
+// RUN:  | FileCheck %s -check-prefixes=CHECK-CPU
+// RUN: %clang -target x86_64-unknown-linux-gnu -fsycl --no-offload-new-driver -fsycl-targets=spir64_x86_64-unknown-unknown -fno-offload-fp32-prec-sqrt -fno-offload-fp32-prec-div %s -### 2>&1 \
+// RUN:  | FileCheck %s -check-prefixes=CHECK-CPU
+// RUN: %clang -target x86_64-unknown-linux-gnu -fsycl --no-offload-new-driver -fsycl-targets=spir64_x86_64-unknown-unknown -fno-offload-fp32-prec-div -fno-offload-fp32-prec-sqrt %s -### 2>&1 \
+// RUN:  | FileCheck %s -check-prefixes=CHECK-CPU
```

Comment:
To reduce the duplication of all of the extensions, can you do something like: `-check-prefixes=CHECK-CPU,CHECK-CPU-FPME`
Here, `CHECK-CPU` does not add the fp_max_error, but is rather checked with a `CHECK-CPU-FPME`